### DISCUSSION TITLE
Obliteration Architecture Design

### DISCUSSION THREAD

**Discussion by bayedieng** [obhq/obliteration#60] [General]
My understanding of how the architecture is presented is that only syscalls will be emulated and not libraries however, if a ps4 executable were to be dynamically-linked wouldn't the PS4 library API have to be implemented as well?

**Comment by ultimaweapon** [comment]
That is the reason why the end users need `PS4UPDATE1.pup.dec`, which contains all of the libraries that are required by a PS4 app.
